### PR TITLE
wic: add wic image for aarch64

### PR DESCRIPTION
**Pull request by muvarov**
add wic image for aarch64 with linux kernel as
EFI start image.

Signed-off-by: Maxim Uvarov <maxim.uvarov@linaro.org>

### CODE REVIEW

File: meta-ledge-bsp/conf/machine/ledge-qemuarm64.conf [hunk 1]
```diff
@@ -23,6 +23,11 @@ UBOOT_DEVICETREE = "qemu_arm64.dtb"
 UBOOT_CONFIG[basic]   = "qemu_arm64_defconfig,,u-boot.bin"
 EXTRA_IMAGEDEPENDS_append = " virtual/bootloader"
 
+IMAGE_FSTYPES += ' wic'
+WKS_FILE = "wic/ledge-qemuarm64-efi-image.wks"
+IMAGE_BOOT_FILES = "aarch_uefi_vfat1.img"
```
Comment (muvarov) [inline meta-ledge-bsp/conf/machine/ledge-qemuarm64.conf]: hm, that is not needed :)

Comment (apalos) [reply meta-ledge-bsp/conf/machine/ledge-qemuarm64.conf]: i'd prefer having a patch for wic images on all architectures anbd not just qemu aarch64

Comment (muvarov) [reply meta-ledge-bsp/conf/machine/ledge-qemuarm64.conf]: Can all other architectures use linux kernel as efi app?

Comment (cpriouzeau) [reply meta-ledge-bsp/conf/machine/ledge-qemuarm64.conf]: It's the good way to make it

**Comment by muvarov** [comment]
New PR created for zeus branch. This one is closed.

**Pull request closed without merging**